device/cig/x86_64-cig_cs6436_54p-r0/Cig-CS6436-54P/port_config.ini

```diff
@@ -0,0 +1,55 @@
+# name          lanes             alias             index          speed
+Ethernet0       8                 Ethernet1/1       0              25000
```

Comment:
Same question as above. Is front panel port start with 0?

Reply:
Updated